projects/bp-gallery/src/views/curator/UnlinkCollectionAction.tsx

```diff
@@ -0,0 +1,71 @@
+import { Icon, Tooltip } from '@mui/material';
+import React, { useCallback, useContext } from 'react';
+import { FlatCollection } from '../../types/additionalFlatTypes';
+import { DialogContext, DialogPreset } from '../shared/DialogWrapper';
+import { useUpdateCollectionMutation } from '../../graphql/APIConnector';
+import { useTranslation } from 'react-i18next';
+
+const UnlinkCollectionAction = ({
+  childCollection,
+  parentCollection,
+}: {
+  childCollection: FlatCollection;
+  parentCollection: FlatCollection;
+}) => {
+  const { t } = useTranslation();
+  const dialog = useContext(DialogContext);
+  const [updateCollection] = useUpdateCollectionMutation({
+    refetchQueries: ['getCollectionInfoById', 'getAllCollections'],
+    errorPolicy: 'all',
+  });
```

Comment:
```suggestion
  const [updateCollection] = useUpdateCollectionMutation({
    refetchQueries: ['getCollectionInfoById', 'getAllCollections'],
  });
```

Yet again due to the global setting ^^